app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -110,6 +111,17 @@ class ThreadFragment : Fragment() {
 
     private var isFavorite = false
 
+    private val onlyThreadIsShown get() = (parentFragment as TwoPaneFragment).isOnlyRightShown()
+
+    // TODO: This is probably too global as a trigger. Find something more refined?
+    private val globalLayoutListener by lazy {
+        OnGlobalLayoutListener {
+            runCatching {
+                binding.toolbar.navigationIcon?.alpha = if (onlyThreadIsShown) 255 else 0
```

Comment:
You can add constant for 255 and 0. Plus, to avoid setting in a loop the same alpha, you could also check the value if it's different from what you want to set ?

Reply:
Done !